### PR TITLE
Added Tailwind CSS into the image-banner

### PR DESCRIPTION
**Pull request by tanujbordikar**
Solves Issue no #108 
Added dynamic Tailwind CSS into the Image Banner file.
 
*Styling Schemas:*
image
 
*Desktop and Mobile View:*
image
image

### CODE REVIEW

File: sections/dev__image-banner.liquid [hunk 1]
```diff
@@ -0,0 +1,134 @@
+<!-- image-banner.liquid -->
+
+<div class="image-banner {{ section.settings.image-banner }}" >
+  {% if section.settings.mobile_image %}
+    <div class="image-banner__image-wrapper flex-basis-1/2">
+      {{ section.settings.mobile_image | image_url: width: 700 | image_tag: class: 'image-banner__image w-full h-auto object-cover' }}
+    </div>
```
Comment (mihir-bombay-studio) [inline sections/dev__image-banner.liquid:7]: All elements in the section should have the dynamic tailwind class connected, to connect the image_tag in liquid you can refer this [issue comment](https://github.com/dear-digital/shopify-theme-skeleton/pull/109#discussion_r1412744457)

File: sections/dev__image-banner.liquid [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  {% endif %}
+
+  <div class="image-banner__text-wrapper {{ section.settings.image-banner__text-wrapper }}">
+    <h2 class="image-banner__title {{ section.settings.image-banner__title }}" style="color: {{ section.settings.color }};">{{ section.settings.title | escape }}</h2>
```
Comment (mihir-bombay-studio) [inline sections/dev__image-banner.liquid:11]: No need for inline styles as tailwind classes are capable of all kind of styling.

File: sections/dev__image-banner.liquid [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    <p class="image-banner__description {{ section.settings.image-banner__description }}">{{ section.settings.description }}</p>
+    {% if section.blocks.size > 0 %}
+      <div class="image-banner__buttons {{ section.settings.image-banner__buttons }}">
+        {% for block in section.blocks %}
+          <a
+            href="{{ block.settings.url }}"
+            class="image-banner__button {{ section.settings.image-banner__button }}"
+            style="background-color: {{ block.settings.background_color }}; color: {{ block.settings.color }}"
```
Comment (mihir-bombay-studio) [inline sections/dev__image-banner.liquid:19]: Remove inline styles

File: sections/dev__image-banner.liquid (hunk 1 above)
Comment (tanujbordikar) [reply sections/dev__image-banner.liquid:7]: Yes, that will be effective to tackle to this issue!

File: sections/dev__image-banner.liquid [hunk 4]
```diff
@@ -0,0 +1,130 @@
+<!-- image-banner.liquid -->
+
+<div class="image-banner {{ section.settings.image-banner }}" >
+  {%- assign image_url_width = section.settings.image_url_width -%}
+  {%- assign image_tag_width = section.settings.image_tag_width -%}
+  {% if section.settings.mobile_image %}
+    <div class="image-banner__image-wrapper">
+      {{ section.settings.mobile_image | image_url: width: image_url_width | image_tag: width: image_tag_width }}
```
Comment (mihir-bombay-studio) [inline sections/dev__image-banner.liquid:8]: We should be able to add tailwind classes to both these elements as well dynamically.

**Comment by tanujbordikar** [comment]
Hii @mihir-bombay-studio Sir, I have made the neccessary changes in this PR, can you please reveiw this PR!

**Pull request merged**